### DISCUSSION TITLE
How to set tempo from MIDI file?

### DISCUSSION THREAD

**Discussion by TravisWoolston** [melanchall/drywetmidi#289] [Q&A] [answered]
How do I set the tempo to what the MIDI file's original tempo is?

Question 2: 
How do I playback without an output device? I'm making a guitar hero type game(lets just call it that) and I don't need the piano. I saw that it's possible in the docs but didn't follow how.

Also this is incredible, thanks for making it public.

**Comment by melanchall** [comment, marked as answer]
Hi, 

> How do I set the tempo to what the MIDI file's original tempo is?

I have no idea what you mean. Set where? If you use `MidiFile.Read`, a `MidiFile` already has "original" Tempo if you didn't change it manually.

> Question 2:
> How do I playback without an output device? I'm making a guitar hero type game(lets just call it that) and I don't need the piano. I saw that it's possible in the docs but didn't follow how.

Just don't pass a device to a method you use to get `Playback`. Please read docs carefully.

Thanks,
Max
